cmd/bedEnrichments/bedEnrichments.go

```diff
@@ -0,0 +1,160 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/interval"
+	"log"
+)
+
+func bedEnrichments(method string, inFile string, secondFile string, noGapFile string, outFile string, verbose int, trimToRefGenome bool) {
+	var err error
+
+	if method != "exact" && method != "normalApproximate" && method != "upperBound" && method != "lowerBound" {
+		log.Fatalf("Error: unknown method. Found: %s.", method)
+	}
+
+	if verbose > 0 {
+		log.Println("Reading bed files.")
+	}
+
+	elementsOne := bed.ReadLite(inFile)
+	elementsTwo := bed.ReadLite(secondFile)
+	noGapRegions := bed.ReadLite(noGapFile)
+
+	if trimToRefGenome {
```

Comment:
This will likely be easier to both read, debug, and test if it were a separate function